src/Symfony/Component/Messenger/Stamp/RedeliveryStamp.php

```diff
@@ -24,12 +24,12 @@ final class RedeliveryStamp implements StampInterface
     private $exceptionMessage;
     private $flattenException;
 
-    public function __construct(int $retryCount, string $exceptionMessage = null, FlattenException $flattenException = null)
+    public function __construct(int $retryCount, string $exceptionMessage = null, FlattenException $flattenException = null, \DateTimeImmutable $redeliveredAt = null)
```

Comment:
this should typehint `DateTimeInterface` since this is what getRedeliveredAt also typehints